mmtrack/core/track/interpolation.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+
+def tracklet_interpolation(results, min_frames=5, max_frames=20):
+    """Interpolate tracklet.
+
+    This function is proposed in
+    "ByteTrack: Multi-Object Tracking by Associating Every Detection Box."
+    `ByteTrack<https://arxiv.org/abs/2110.06864>`_.
+
+    Args:
+        results (ndarray): With shape (N, 7). Each row denotes
+            (frame_id, track_id, x1, y1, x2, y2, score)
+        min_frames (int, optional): The minimum frames of a tracklet that will
+            be interpolated. Defaults to 5.
+        max_frames (int, optional): The maximum frames of a tracklet that will
+            be interpolated. Defaults to 20.
+
+    Returns:
+        ndarray: The interpolated results with shape (N, 7). Each row denotes
+            (frame_id, track_id, x1, y1, x2, y2, score)
+    """
+    interpolation_results = []
+    max_track_id = int(np.max(results[:, 1]))
+    min_track_id = int(np.min(results[:, 1]))
+
+    # perform interpolation for each tracklet
+    for track_id in range(min_track_id, max_track_id + 1):
```

Comment:
It would be great to first write a function for "interpolate_tracklet", which interpolates only a tracklet, then another function for some tracklets.